### DISCUSSION TITLE
Can you set `loading=lazy` as a default for all images?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3803] [Q&A] [answered]
Simple goal: Would like to have all images in mkdocs load lazily with the `loading=lazy` attribute, so that I don't have to append the `{ loading=lazy }` attribute format to all image links.

Is this possible? What are the required steps?

**Comment by squidfunk** [comment, marked as answer]
Not yet possible. Could be implemented as a Markdown extension or MkDocs plugin. It's not specific to Material for MkDocs.
